.nf-test/tests/a5db341bb7d3e73dbb7674a87b560ca/meta/nextflow.log

```diff
@@ -0,0 +1,74 @@
+Mar-14 11:18:08.369 [main] DEBUG nextflow.cli.Launcher - $> nextflow -quiet -log /Users/agopalan/cckp-toolkit-workflow/.nf-test/tests/a5db341bb7d3e73dbb7674a87b560ca/meta/nextflow.log run /Users/agopalan/cckp-toolkit-workflow/main.nf -c /Users/agopalan/cckp-toolkit-workflow/nextflow.config -params-file /Users/agopalan/cckp-toolkit-workflow/.nf-test/tests/a5db341bb7d3e73dbb7674a87b560ca/meta/params.json -ansi-log false -with-trace /Users/agopalan/cckp-toolkit-workflow/.nf-test/tests/a5db341bb7d3e73dbb7674a87b560ca/meta/trace.csv -w /Users/agopalan/cckp-toolkit-workflow/.nf-test/tests/a5db341bb7d3e73dbb7674a87b560ca/work
```

Comment:
make sure you delete this and add the folders/files that nextflow/nf-test generates to your gitignore